app/move/controllers/create/base.js

```diff
@@ -0,0 +1,31 @@
+const FormWizardController = require('../form-wizard')
+
+class CreateBaseController extends FormWizardController {
+  middlewareChecks() {
+    super.middlewareChecks()
+    this.use(this.checkCurrentLocation)
+  }
+
+  middlewareLocals() {
+    super.middlewareLocals()
+    this.use(this.setCancelUrl)
+  }
+
+  setCancelUrl(req, res, next) {
+    res.locals.cancelUrl = res.locals.MOVES_URL
+    next()
+  }
+
+  checkCurrentLocation(req, res, next) {
+    if (!req.session.currentLocation) {
+      const error = new Error(
+        'Current location is not set. Check environment variable is correctly set.'
```

Comment:
I'm wondering if this is the correct place to check this? 
If we need the application to work with this environment variable maybe we should be checking in one place on startup of the app? 
Maybe something that loops through a required list of `envs` and checks they are set in `process.env`?

Reply:
In fact, this may be redundant with the recent set of changes to how locations are handled.

I'll address this in a separate PR.

Reply:
Have updated this in #207